### PR TITLE
Python27

### PR DESCRIPTION
**Pull request by daniviga**
Tests and demos are now run with both Python 2.7 and Python 3.5. First tests are run in parallel and then the demos.

### CODE REVIEW

**Comment by daniviga** [comment]
Should work now

File: .travis.yml [hunk 1]
```diff
@@ -7,33 +7,59 @@ notifications:
...
-        - OQ_DISTRIBUTE=celery nosetests --with-doctest -xvs -a'!slow' openquake.server
-        - OQ_DISTRIBUTE=celery nosetests --with-doctest -xvs -a'!slow' openquake.calculators
-        - OQ_DISTRIBUTE=celery nosetests --with-doctest -xvs -a'!slow' openquake.risklib
-        - OQ_DISTRIBUTE=celery nosetests --with-doctest -xvs -a'!slow' openquake.commonlib
-        - OQ_DISTRIBUTE=celery nosetests --with-doctest -xvs -a'!slow' openquake.commands
+        - nosetests --with-doctest -xvs -a'!slow' openquake.engine
+        - nosetests --with-doctest -xvs -a'!slow' openquake.server
+        - nosetests --with-doctest -xvs -a'!slow' openquake.calculators
+        - nosetests --with-doctest -xvs -a'!slow' openquake.risklib
+        - nosetests --with-doctest -xvs -a'!slow' openquake.commonlib
```
Comment (micheles) [inline .travis.yml]: I would try to do this in a single line
```
 nosetests --with-doctest -xvs -a openquake
```
also including the slow tests, so we do not forget the hmtk or directories that may be added in the future

File: .travis.yml [hunk 2]
```diff
@@ -7,33 +7,59 @@ notifications:
...
+      - OQ_DISTRIBUTE=celery
+      - PYTHON=3.5
+    python: "3.5"
+    script:
+        - nosetests --with-doctest -xvs -a'!slow' openquake.engine
+        - nosetests --with-doctest -xvs -a'!slow' openquake.server
+        - nosetests --with-doctest -xvs -a'!slow' openquake.calculators
+        - nosetests --with-doctest -xvs -a'!slow' openquake.risklib
+        - nosetests --with-doctest -xvs -a'!slow' openquake.commonlib
+        - nosetests --with-doctest -xvs -a'!slow' openquake.commands
```
Comment (micheles) [inline .travis.yml]: Same comment as before

File: .travis.yml (hunk 1 above)
Comment (daniviga) [reply .travis.yml]: Agreed

**Pull request merged**